flybywiresim/a32nx
# fix(vfx): contrails now depend on engine actually running

Fixes #8024

## Summary of Changes


Contrails now depend on the engine N2 to be above 25% so they turn off when the engine is turned off.

## Screenshots (if necessary)




## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


1. Find situation where contrails are visible.
2. Turn off one engine
3. See contrail turn off as engine N2 goes below 25%
4. Turn engine back on 
5. See contrails return after N2 goes above 25% 


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
